File: app/src/main/java/com/vikingsen/cheesedemo/model/webservice/dto/CommentResponse.java
package com.vikingsen.cheesedemo.model.webservice.dto;


public class CommentResponse {
    private String guid;
    private int status;

    public String getGuid() {
        return guid;
    }

    public void setGuid(String guid) {
        this.guid = guid;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }
}

File: app/src/main/kotlin/com/vikingsen/cheesedemo/model/webservice/dto/CommentResponse.kt
package com.vikingsen.cheesedemo.model.webservice.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class CommentResponse(val guid: String, val status: Int) {
    // Code Created
    val isSuccessful: Boolean
        get() = status == 201
}
